Felienne/hedy
# [UI IMPROVEMENT] Flips debug button for RTL languages

**Description**
In this PR we make a minor QoL fix by flipping the debug button when on a RTL language. We also improve some minor styling on the debug container as it was only styled for LTR support. Funny note: there is no "fast-rewind" button so we had to rotate the original icon 180 degrees.

**Fixes**
This PR fixes #2759.

**How to test**
Select a RTL language such as Arabic and verify that the debug button is now correctly pointing to the left side of the screen. Also verify that the debug container is aligned correctly (it wasn't before when on RTL).
